context/app/static/js/components/entity-search/sidebar/DatasetConfigureSearchTable/hooks.js

```diff
@@ -61,6 +61,7 @@ function useFieldGroups() {
 
 function buildFieldConfigs(fieldNames) {
   return fieldNames.reduce((acc, fieldName) => {
+    // Version must be removed before creating field configs because it does not have a type defined in ingest-validation-tools docs.
```

Comment:
It's an integer, and I could provide it in the field list... but my sense is that it's really only useful when supplying data: Downstream, it shouldn't matter what schema version was used. Anyway: push for this if that doesn't seem right to you.